uftrace.c

```diff
@@ -825,6 +828,16 @@ static int parse_option(struct opts *opts, int key, char *arg)
 		opts->diff_policy = arg;
 		break;
 
+	case OPT_format:
+		if (!strcmp(arg, "html")) {
+			format_mode = FORMAT_HTML;
+			if (opts->color == COLOR_AUTO)
+				opts->color = COLOR_ON;
+		}
+		else
+			format_mode = FORMAT_NORMAL;
```

Comment:
I'm not sure about this part.  This will accept any invalid strings..  should be better if it shows a warning.

Reply:
I put a warning message in that case.